docs/source/tutorials/submission.ipynb

```diff
@@ -0,0 +1,233 @@
+{
```

Comment:
<ul><li>Same here it would be good to replace "you" with "user"</li><li>To run RP an RP application -&gt; did you mean "To run RP and RP application"</li><li> find out how many  -&gt;  determine how many </li><li>in number 4 could be good to explain what is what.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2874/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>